test/integration/ferc1_eia_train_test.py

```diff
@@ -140,22 +153,23 @@ def test_validate_override_fixes(
         start_date=min(eia_ferc1.report_date),
         end_date=max(eia_ferc1.report_date),
     )
-    validate_override_fixes(
-        validated_connections=test_df,
-        ppe=plant_parts_eia,
-        eia_ferc1=eia_ferc1,
-        training_data=eia_ferc1_training_data_restricted,
-        expect_override_overrides=False,
-        allow_mismatched_utilities=False,
-    )
+    with expectation:
+        validate_override_fixes(
+            validated_connections=test_df,
+            ppe=plant_parts_eia,
+            eia_ferc1=eia_ferc1,
+            training_data=eia_ferc1_training_data_restricted,
+            expect_override_overrides=False,
+            allow_mismatched_utilities=False,
+        )
 
 
 def test_generate_all_override_spreadsheets(
     pudl_engine: sa.Engine,  # Required to ensure that the data is available.
 ):
     """Test the genation of the override spreadsheet for mapping FERC-EIA records."""
     # Get data tables directly
-    plant_parts_eia = get_parquet_table("out_eia__yearly_plant_parts").reset_index()
+    plant_parts_eia = get_parquet_table("out_eia__yearly_plant_parts")
```

Comment:
No point in resetting the index on a dataframe we just read out of Parquet -- this is a vestige from a bygone era.